src/panels/lovelace/cards/hui-picture-elements-card.ts

```diff
@@ -94,6 +111,18 @@ class HuiPictureElementsCard extends LitElement implements LovelaceCard {
     `;
   }
 
+  private _evalConditions(elementConfig: LovelaceElementConfig): boolean {
```

Comment:
We shouldn't built this into the picture elements card, instead it should be a conditional element, like the conditional card.

Reply:
I'm pretty sure you can already do it with the existing conditional card; only caveat is that you would use a "custom" element for the card. e.g. 

type: custom:hui-icon-element

Reply:
I think I know what @balloob means. Will try.
@iantrich - the idea is to do this at the element level within picture elements. I'll check if hui-conditional-card would work. 
Anyway something that is not part of the card will be missed by many users and it's a shame IMHO.

Reply:
@iantrich so I tried this (see below) and off course it fails since picture-elements inner elements are not cards. Am I missing something?
Later once I have time I will go with what Paulus suggested to wrap it in a conditional element that hosts other elements.

```
image: 'https://home.consiv.com/local/garden.jpg'
type: picture-elements
elements:
  - type: custom:hui-conditional-card
    conditions:
      - entity: input_boolean.bool1
        state: on
    card:
    - type: state-label
      entity: sensor.solar_angle
```

Reply:
Done. Refactored to be a new conditional element that allows sub elements.